src/commands/generate.ts

```diff
@@ -48,32 +53,73 @@ const generatePrompt = (templates: string[], load: any, options: OptionValues) =
         if (/^([A-Za-z\-\_\d])+$/.test(input)) return true;
         else return 'Project name may only include letters, numbers, underscores and hashes.';
       }
+    },
+    {
+      name: 'app-confirm',
+      type: 'confirm',
+      default: false,
+      message: 'Do you want to generate template for UI app?'
+    }
+  ];
+
+  const APP = [
+    {
+      name: 'app-choice',
+      type: 'list',
+      message: 'What UI template would you like to generate?',
+      choices: uiTemplates
     }
   ];
 
   inquirer.prompt(QUESTIONS).then((answers) => {
     const projectChoice = answers['project-choice'];
     const projectName = answers['project-name'];
+    const appConfirm = answers['app-confirm'];
 
-    if (fs.existsSync(path.resolve(projectName))) {
-      fs.rmSync(path.resolve(projectName), { recursive: true, force: true });
-    }
+    if (appConfirm == true) {
+      inquirer.prompt(APP).then(() => {
+        const appChoice = answers['app-choice'];
+        if (fs.existsSync(path.resolve(projectName))) {
```

Comment:
you have basically the same code inside if and else statements, i would move it to a separated method and parametrize `exec` method

Reply:
Done!